nautobot/core/management/commands/migrate.py

```diff
@@ -1,8 +1,105 @@
+        """
+        Enhanced version of Django's built-in callback.
+
+        Differences in behavior:
+            - Measures and reports elapsed time whenever verbosity >= 1 instead of requiring verbosity >= 2.
+            - Measures and reports affected record counts and elapsed time per record where applicable
+            - Aligns output to 80-character terminal width in most cases
+
+        Examples:
+            # nautobot-server migrate --verbosity=1
+            Running migrations:
+              Applying dcim.0065_controller_capabilities_and_more...         OK    (   0.1s)
+                Affected dcim.controller                              11 rows  0.007s/record
+                Affected dcim.controllermanageddevicegroup            35 rows  0.002s/record
+              Applying wireless.0001_initial...                              OK    (   0.3s)
+                Affected wireless.controllermanageddevicegroupradioprofileassignment        0 rows
+                Affected wireless.controllermanageddevicegroupwirelessnetworkassignment        0 rows
+                Affected wireless.radioprofile                         0 rows
+                Affected wireless.supporteddatarate                    0 rows
+                Affected wireless.wirelessnetwork                      0 rows
+              Applying dcim.0066_controllermanageddevicegroup_radio_profi... OK    (   0.1s)
+                Affected dcim.controllermanageddevicegroup            35 rows  0.002s/record
+
+            # nautobot-server migrate -v=1 extras 0100
+            Operations to perform:
+              Target specific migration: 0100_fileproxy_job_result, from extras
+            Running migrations:
+              Rendering model states...                                      DONE  (  19.1s)
+              Unapplying ipam.0052_alter_ipaddress_index_together_and_mor... OK    (   0.1s)
+                Affected ipam.ipaddress                               75 rows  0.001s/record
+                Affected ipam.prefix                                 184 rows  0.000s/record
+                Affected ipam.vrf                                     20 rows  0.004s/record
+              Unapplying ipam.0051_added_optional_vrf_relationship_to_vdc... OK    (   0.1s)
+                Affected ipam.vrf                                     20 rows  0.006s/record
+                Affected ipam.vrfdeviceassignment                    140 rows  0.001s/record
+              Unapplying virtualization.0030_alter_virtualmachine_local_c... OK    (   0.3s)
+                Affected virtualization.virtualmachine                 0 rows
+                Affected virtualization.vminterface                    0 rows
+              Unapplying virtualization.0029_add_role_field_to_interface_... OK    (   0.1s)
+                Affected virtualization.vminterface                    0 rows
+        """
+        if self.verbosity < 1:
```

Comment:
Let's make sure the expensive parts of the migration are only done on verbosity 2.